Contest_Archive_Format.md

```diff
@@ -73,15 +73,27 @@ Optionally one could create a Shallow Archive by not storing the files, in
 which case the URLs must be valid. This could be useful in some cases where
 the size of the archive matters.
 
-In some cases it could make sense to merge multiple API sources (of the same
-contest) in a single archive. One example of this would be a contest that was
-running with a primary and [shadow](ccs_system_requirements#shadow-mode) CCS.
-Typically in such cases, most of the data is identical (or at least the
-differences are irrelevant), so only the data that differs (in relevant ways)
-needs to be stored. If there is an additional source for `<endpoint>` it is
-stored as above, but wherever `<endpoint>` is used in the path, instead use
-`<endpoint>_<system>` where `<system>` is a unique name for the additional
-source.
+### Multiple systems
+
+Some contests are run with multiple systems that provide a Contest API.
+One example of this is a contest run with a primary and
+[shadow](ccs_system_requirements#shadow-mode) CCS. Typically in these cases
+there is relationship between the systems or a lot of common data, so it
+makes sense to merge the data into a single archive.
+
+Data from the primary system (usually the primary CCS) should continue to
+be stored as described in this document. Each additional system should
+store its data using the same format but in a unique sub-folder
+`systems/<system>`, named after the system that produced it.
```

Comment:
Maybe `other-systems`? I'm not sure it's better, but it does make it more explicit that the root directory already contains a full archive and that these are additional ones.

Reply:
I don't love this, but the only downside I could come up with 'it's longer' so I made the change. :)